scripts/pulsars_in_fov.py

```diff
@@ -171,21 +173,27 @@ def find_pulsars_in_fov(obsid, psrbeg, psrend):
 
     # Find all of the Fermi candidates
     #-----------------------------------------------------------------------------------------------------------
-    temp = get_sources_in_fov(obsid, 'Fermi', fwhm)
-    pulsar_name_list = pulsar_name_list + temp[0]
-    pulsar_pointing_list = pulsar_pointing_list + temp[1]
+    fermi_list = get_sources_in_fov(obsid, 'Fermi', fwhm)
+    pulsar_search_name_list = pulsar_search_name_list + fermi_list[0]
+    pulsar_search_pointing_list = pulsar_search_pointing_list + fermi_list[1]
 
     # Find all of the points of interest candidates
     #-----------------------------------------------------------------------------------------------
-    temp = get_sources_in_fov(obsid, 'POI', fwhm)
-    pulsar_name_list = pulsar_name_list + temp[0]
-    pulsar_pointing_list = pulsar_pointing_list + temp[1]
```

Comment:
not using temp <3

Reply:
You highlighted something I removed preemptively to keep you happy!